src/System.Data.SqlClient/tests/ManualTests/SQL/SqlNamedPipesTest/SqlNamedPipesTest.cs

```diff
@@ -69,10 +69,6 @@ public static void InvalidConnStringTest()
             // Nothing but slashes
             builder.DataSource = @"np:\\\\\";
             OpenBadConnection<SqlException>(builder.ConnectionString, invalidConnStringError);
-
-            // Empty string
-            builder.DataSource = "np:";
```

Comment:
I removed this test case because its not supposed to result in a failure. 
np: is the same as np:localhost and this shouldn't be flagged as a Connection String Error.